tabs/pid_tuning.js

```diff
@@ -741,53 +770,126 @@ TABS.pid_tuning.initialize = function (callback) {
         }
 
         // Getting the DOM elements for curve display
-        var rcCurveElement = $('.rate_curve canvas').get(0);
-        var curveContext = rcCurveElement.getContext("2d");
+        var rcCurveElement              = $('.rate_curve canvas').get(0),
```

Comment:
I'd rather we used explicit names for the individual canvas elements. Referring to them by index will make later extension prone to break.